### PR TITLE
Materialize css navbar added in every page

### LINKED ISSUE #15: Navigation Bar
Try to have a microanimation when we hover over one tab then a lamp enlights it

### PR DESCRIPTION
**Pull request by Adyasha8105**
**Fixes #15** 
- I have converted the navbar using materialise css .
- The project page color has changed.
- Made a common CSS and script for all pages (navbar) i.e style.css & main.js

**Screenshot:-**
image
image

**Next:- Home page**
@smaranjitghose @anushbhatia Please check it :).

### CODE REVIEW

**Comment by anushbhatia** [comment]
Please try to use materializecss its new for you but it gives a minimal UI and better look with animations if possible. If you have  any issues related to the documentation or implementation feel free to contact me on slack.

**Comment by smaranjitghose** [comment]
When we hover over into one of the tabs in the navbar, that line below it should not appear as an underline, make it broader and a little lower

navbar

**Comment by smaranjitghose** [comment]
Please fix the merge conflicts

File: education.html [hunk 1]
```diff
@@ -6,28 +6,44 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <title>Education</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />  
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />  
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
```
Comment (sakshi1499) [inline education.html:9]: Don't remove and add the same line again please

File: education.html [hunk 2]
```diff
@@ -6,28 +6,44 @@
...
+      </a>
+
+      <ul class="right hide-on-med-and-down">
         <li><a href="./index.html">Home</a></li>
         <li><a href="#">About</a></li>
         <li><a href="./experience.html">Experience</a></li>
         <li><a href="./projects.html">Projects</a></li>
-        <li><a href="./education.html">Education</a></li>
         <li><a href="#">Contact</a></li>
+        <li><a href="./education.html">Education</a></li>
```
Comment (sakshi1499) [inline education.html:32]: Why was this change made, you've done it on all the pages. In fact I feel that the Contact tab should be at the right most side

File: education.html (hunk 1 above)
Comment (Adyasha8105) [reply education.html:9]: oh sorry I didn't mean to.

File: education.html (hunk 2 above)
Comment (Adyasha8105) [reply education.html:32]: okay, then I have done this according to the design of home page in xd file. Should I change it?

**Comment by Adyasha8105** [comment]
@sakshi1499 Should I change it or we will later think about it?

**Comment by smaranjitghose** [comment]
@Adyasha8105 

Please Nav Bar as:
- Home
- Experience
- Projects
- Education
- Blog

We will put the contact feature in the footer and the about will be included in the home page itself

**Pull request merged**